l10n/bundle.l10n.json

```diff
@@ -8,11 +8,10 @@
   "Unmatched number of curly braces for expression": "Unmatched number of curly braces for expression",
   "Select properties for element '{0}'": "Select properties for element '{0}'",
   "Visual Element added": "Visual Element added",
-  "Enter Python executable path. `python` value will be used if the field is empty": "Enter Python executable path. `python` value will be used if the field is empty",
-  "This path is used to locate the visual element descriptors file": "This path is used to locate the visual element descriptors file",
+  "Can't find visual element descriptors file with the provided environment": "Can't find visual element descriptors file with the provided environment",
```

Comment:
Isn't there a dot missing at the end?
That would depend on the context, that I don't see right now...

Reply:
you sure you want dots. i can add dots in all commands